### PR TITLE
Add `GetMapRequest`.

### PR DESCRIPTION
**Pull request by siarheishut**

### CODE REVIEW

File: src/request_parser.cpp [hunk 1]
```diff
@@ -192,4 +194,17 @@ std::optional<GetRouteRequest> ParseGetRouteRequest(json::Dict dict) {
...
+  if (id == dict.end())
+    return std::nullopt;
+  if (!id->second.IsInt())
+    return std::nullopt;
+
+  GetMapRequest mr;
+  mr.id = id->second.AsInt();
+
+  return mr;
+}
```
Comment (foxfrikses) [inline src/request_parser.cpp:209]:
```suggestion
  auto id = dict.find("id");
  if (id == dict.end() || !id->second.IsInt())
    return std::nullopt;
  return GetMapRequest{.id = id->second.AsInt()};
}
```

Comment (siarheishut) [reply src/request_parser.cpp:209]: Done.

File: src/request_parser.cpp [hunk 2]
```diff
@@ -192,4 +194,12 @@ std::optional<GetRouteRequest> ParseGetRouteRequest(json::Dict dict) {
 
   return gr;
 }
+
+std::optional<GetMapRequest> ParseGetMapRequest(json::Dict dict) {
```
Comment (foxfrikses) [inline src/request_parser.cpp:198]: I can't find tests for this function. Weren't you going to add them here as well?

Comment (siarheishut) [reply src/request_parser.cpp:199]: Added.

**Review by foxfrikses — APPROVED** [review]
LGTM

**Pull request merged**